com.ibm.streamsx.metrics/build.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2011, 2013
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with
+# IBM Corp.
+-->
+<project name="com.ibm.streamsx.metrics" basedir="." default="all">
+
+	<property environment="env" />
+	<fail unless="env.STREAMS_INSTALL" message="STREAMS_INSTALL not set."/>
+	<available file="${basedir}/pom.xml" property="pomset"/>
+	<property name="streams.install" value="${env.STREAMS_INSTALL}" />
+
+	<property name="maven.bin" value="${env.M2_HOME}/bin/mvn" />
+	<property name="ext.downloads.dir" value="opt/downloaded" />
+	<property name="src.dir" value="impl/java/src" />
+	<property name="build.dir" value="impl/java/classes" />
+	<property name="override.dir" value="opt/local_override" />
+	<property name="impl.lib.dir" value="impl/lib" />
+	<property name="lib.dir" value="lib" />
+	<property name="test.run.dir" value="tests" />
+	<property name="test.build.dir" value="${test.run.dir}/bin" />
+	
+	<target name="init">
+		<mkdir dir="${build.dir}" />
+		<mkdir dir="${override.dir}" />
+		<mkdir dir="${lib.dir}" />
+		<mkdir dir="${impl.lib.dir}" />
+		<mkdir dir="${ext.downloads.dir}" />
+	</target>
+
+	<target name="clean">
+		<delete dir="${build.dir}" />
+		<delete dir="${impl.lib.dir}" />
+		<delete dir="output"/>
+		<delete dir="${test.run.dir}"/>
+		<delete dir="${test.build.dir}"/>
+	</target>
+
+
+	<target name="distclean" depends="clean">
+		<delete dir="${ext.downloads.dir}" />
+	</target>
+
+
+	<path id="cp.streams">
+		<pathelement location="${streams.install}/lib/com.ibm.streams.operator.samples.jar" />
+	</path>
+
+
+	<target name="all" depends="jar">
+	 <exec executable="${streams.install}/bin/spl-make-toolkit">
+           <arg value="--directory"/>
+           <arg value="${basedir}"/>
+	 </exec>
+	</target>
+
+	
+	<target name="compile" depends="init,maven-deps">
+		<javac srcdir="${src.dir}" destdir="${build.dir}" debug="true"
+			includeantruntime="no">
+			<classpath>
+				<path refid="cp.streams" />
+			</classpath>
+		</javac>
+	</target>
+
+	<target name="jar" depends="compile">
+		
+		<jar destfile="${impl.lib.dir}/com.ibm.streamsx.hbase.jar" basedir="${build.dir}" includes="com/ibm/streamsx/hbase/**" filesonly="true">
+	
+		</jar>
+		<delete dir="${build.dir}" />
+	</target>
+
+	<!-- Downloads libraries using maven -->
+	<target name="maven-deps" depends="init,nomaven" if="env.M2_HOME">
+		<fail unless="pomset" message="There must be a pom.xml file in com.ibm.streamsx.hbase in order to use maven to download necessary dependencies.  Three sample pom files are included: pom-v094.xml (hbase 0.94, hadoop 1),pom-v096-hadoop1.xml (hbase 0.96, hadoop 1), and  pom-v096-hadoop2.xml (hbase 0.96, hadoop 2).  Please rename the correct one for your install to pom.xml"/> 
```

Comment:
Do you need streamsx.hbase? I assume that you copied this file from another repo that also copied it from another one. Please cleanup this file, so also messages and comments are correct.